python/lsst/meas/algorithms/matcherSourceSelector.py

```diff
@@ -162,4 +177,38 @@ def _isUsable(self, source):
             and not source.get(self.fluxFlagKey) \
             and self._goodSN(source)
 
+    def _isUsable_vector(self, sourceCat):
```

Comment:
Don't know why this didn't crash the code in the first place. The duplicates have been deleted. I had remembered to run pep8 on the meas_astrom part of this ticket but forgot to do it here before the pull request. I have run a linter on the code as of the most recent commit.